core/frontend/src/extension/providers/ExtensionServiceClient.ts

```diff
@@ -72,12 +72,13 @@ export class ExtensionClient {
   }
 
   /**
-   * Gets information on extensions. If extensionName is undefined, will return all extensions in the context.
+   * Gets information on extensions. If extensionName is undefined, will return all extensions in the iTwinId.
    * If it's defined, will return all versions of that extension.
-   * @param iTwinId Context Id
+   * If iTwinId is undefined, will default to public.
    * @param extensionName Extension name (optional)
+   * @param iTwinId iTwin Id (optional)
    */
-  public async getExtensions(accessToken: AccessToken, iTwinId: string, extensionName?: string): Promise<ExtensionMetadata[]> {
+  public async getExtensions(accessToken: AccessToken, extensionName?: string, iTwinId = Guid.empty): Promise<ExtensionMetadata[]> {
```

Comment:
this is prob fine since i guess no one will have used these apis yet, but i wonder if you should leave the ordering of args as is or if we switch to an object as input. 

same comment method for below

Reply:
For the method below, I wanted to move iTwinId to the last argument since its the only optional argument. And I wanted the method below and the one above to match.
But, not against the idea of switching to object input, maybe we wait for ExtensionClient to be moved into its own repo? (The only user of ExtensionClient _right now_ is ServiceExtensionProvider, its not even exported)